engine/apps/alerts/models/alert_group.py

```diff
@@ -527,6 +531,7 @@ def telegram_permalink(self) -> typing.Optional[str]:
     def permalinks(self) -> Permalinks:
```

Comment:
does this affect the output of `yarn generate-types`? (wondering if the `Permalinks` type needs to be updated here?)

Reply:
good point! updating and pushing :+1: